db/migrate/20220520173939_add_replacement_procedure_id_to_procedures.rb

```diff
@@ -0,0 +1,5 @@
+class AddReplacementProcedureIdToProcedures < ActiveRecord::Migration[6.1]
+  def change
+    add_column :procedures, :replacement_procedure_id, :integer
```

Comment:
si on garde ce champs plutot que parent_procedure_id ; il faudrait en faire une reference histoire d'être du bon coté de la barrière pour l'intégrité des données
```suggestion
    add_column :procedures, :replacement_procedure_id, :bigint
```